lib/ics23/ics23.nix

```diff
@@ -9,6 +21,6 @@
       };
     in
     {
-      checks = ics23TestSuite.checks;
+      inherit (ics23TestSuite) checks;
```

Comment:
FYI - this can actually cause issues with infinite recursion. that's why we don't use inherit here in some of the rust nix files